libopenage/renderer/stages/camera/manager.cpp

```diff
@@ -16,7 +16,7 @@ CameraManager::CameraManager(const std::shared_ptr<renderer::camera::Camera> &ca
 	zoom_motion_direction{static_cast<int>(ZoomDirection::NONE)},
 	move_motion_speed{0.2f},
 	zoom_motion_speed{0.05f},
-	camera_boundaries({X_MIN, X_MAX, Z_MIN, Z_MAX}) {
+	camera_boundaries{X_MIN, X_MAX, 0.0f, 0.0f, Z_MIN, Z_MAX} {
```

Comment:
Initializing to `0.0f` for min/max is not that great, since that will probably move the camera into the terrain. That's why it is initialized to `10.0f` by default.

Reply:
I guess using the suggestion in https://github.com/SFTtech/openage/pull/1691#discussion_r1823620198 would solve this.